apps/laboratory/tests/multichain/multichain-ethers-solana-siwe.spec.ts

```diff
@@ -53,5 +56,5 @@ test('it should switch networks and sign siwe', async () => {
 test('it should switch to Solana and validate chain title', async () => {
   const chainName = 'Solana'
   await modalPage.switchNetwork(chainName)
-  await modalValidator.expectSwitchChainView(chainName.toLocaleLowerCase())
+  await modalValidator.expectSwitchedNetworkOnNetworksView(chainName)
```

Comment:
Again, if the expected behavior is to keep switch chain view open after the chain is switched, this test was wrong.

Reply:
how was it passing before?